packages/message-forwarder/src/forwardMessage/forwardMessage.ts

```diff
@@ -21,30 +21,33 @@ const forwardMessage = async (message: string, client: Client): PromiseResult<vo
     throw new Error(`Unsupported destination type: "${destinationType}"`)
   }
 
-  const aho = parseAhoXml(message)
-  if (isError(aho)) {
-    throw aho
+  const maybeAHO = parseAhoXml(message)
+  if (isError(maybeAHO)) {
+    throw maybeAHO
+  }
+  const correlationId = maybeAHO.AnnotatedHearingOutcome.HearingOutcome.Hearing.SourceReference.UniqueID
+
+  if (destinationType === DestinationType.MQ) {
```

Comment:
Move this up so that if we're in MQ mode, we don't bother checking Conductor (it doesn't exist in legacy mode in CI, which is what breaks everything)